src/main/java/org/folio/rest/impl/PatronServicesResourceImpl.java

```diff
@@ -74,7 +75,7 @@ public void getPatronAccountById(String id, boolean includeLoans,
                 .thenApply(this::verifyAndExtractBody)
                 .thenApply(body -> addLoans(account, body, includeLoans));
 
-            final CompletableFuture<Account> cf2 = httpClient.request("/circulation/requests?limit=" + getLimit(includeHolds) + "&query=%28requesterId%3D%3D" + id + "%20and%20requestType%3D%3DHold%20and%20status%3D%3DOpen%2A%29", okapiHeaders)
+            final CompletableFuture<Account> cf2 = httpClient.request("/circulation/requests?limit=" + getLimit(includeHolds) + "&query=%28requesterId%3D%3D" + id + "%20and%20status%3D%3DOpen%2A%29", okapiHeaders)
```

Comment:
Similar to what we discussed for SIP, I don't think we need to manually encode the URL here, right? If so we might need a story to remove the encoding characters for mod-patron.

Reply:
Why are we making this change, we don't need to query by requestType anymore?

Reply:
Yeah, that is old school pre-encoded stuff. We'll have to verify we don't need it when using the RMB client before it is removed... I could have sworn we manually encoded for a reason. Maybe not... 😄 

As far as the request type, as mentioned in the description, we don't want to limit to this query to just holds, since we'll be placing pages and possibly recalls. They will not be returned if we limit the results to `requestType==Hold`.

Reply:
Ah, I skipped the description. I need some sleep...